assets/src/edit-story/components/button/index.js

```diff
@@ -27,6 +27,8 @@ import RedoIcon from './icons/redo.svg';
 import LeftArrowIcon from './icons/arrow_left.svg';
 import RightArrowIcon from './icons/arrow_right.svg';
 import GridViewIcon from './icons/grid_view.svg';
+import CloseIcon from './icons/close.svg';
```

Comment:
These should use React component after #244

Reply:
Yep 👍  the branch hasn't been rebased yet